src/client/client.ts

```diff
@@ -148,6 +221,109 @@ export default class CTProtoClient<AuthRequestPayload, AuthResponsePayload, ApiR
     this.init();
   }
 
+  /**
+   * This method sends file requests
+   *
+   * @param type - available type of requests
+   * @param file - file to send
+   * @param payload - available request payload
+   */
+  public async sendFile(
+    type: ApiRequest['type'],
+    file: Buffer,
+    payload: ApiRequest['payload']
+
+  ): Promise<ApiResponse['payload']> {
+    return new Promise( resolve => {
+      const callback = (response: ApiResponse['payload']): void => {
+        resolve(response);
+      };
+
+      const fileId = MessageFactory.createFileId();
+
+      const fileSize = file.length;
+
+      /**
+       * Calculate number of chunks
+       */
+      const chunks = Math.ceil(file.length / this.bufferLimit);
+
+      /**
+       * Create new uploading file to save data for file uploading
+       */
+      this.uploadingFiles.push({ id: fileId,
+        chunks: file,
+        sendingChunk: 0,
+        cb: callback,
+        resendTimes: 0,
+      });
+
+      const message = MessageFactory.createMessageForChunk(type, payload, chunks, fileSize);
+
+      this.sendChunk( message, fileId );
+    });
+  }
+
+
+  /**
+   * This method sends one chunk of the uploading file
+   *
+   * @param message - additional info for chunk
+   * @param fileId - id of sending file
+   */
+  public sendChunk(message: string, fileId: string): void {
+    const uploadingFile = this.getUploadingFileById(fileId);
+
+    if (!uploadingFile) {
+      return;
+    }
+
+    /**
+     * Getting chunk by slicing file by the chunk number and buffer limit
+     */
+    const chunk = uploadingFile.chunks.slice( uploadingFile.sendingChunk * this.bufferLimit, this.bufferLimit + this.bufferLimit * uploadingFile.sendingChunk );
+
+    const meta = this.makeMetaData(uploadingFile.sendingChunk, chunk.length);
```

Comment:
please, add a jsdoc describing what is a "meta data". Because this method has a `message` parameter that is described like "additional info for chunk". Sound similar with "meta data". So what is the difference?